packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -179,8 +215,39 @@ const ReflectionCard = (props: Props) => {
   const clearError = () => {
     onCompleted()
   }
+
+  const handleClickSpotlight = (e: MouseEvent) => {
+    e.stopPropagation()
+    const el = reflectionRef.current
+    if (openSpotlight && el) {
+      openSpotlight(reflectionId, reflectionRef)
+      setSelectedForSpotlight(true)
+    }
+  }
+
+  useLayoutEffect(() => {
+    // the same reflection card can exist in the kanban and spotlight
+    // when the spotlight closes, make the kanban reflection visible again
+    if (!inSpotlight && selectedForSpotlight) {
+      setSelectedForSpotlight(false)
```

Comment:
Initially I implemented this by editing the ref styles directly but I think using state here is easier to follow